ethcore/src/engines/mod.rs

```diff
@@ -89,6 +89,8 @@ pub enum EngineError {
 	InvalidEngine,
 	/// Requires signer ref, but none registered.
 	RequiresSigner,
+	/// Missing Parent Epoch
+	MissingParent,
```

Comment:
Does it make sense to include the parent hash in the error?

Reply:
> Does it make sense to include the parent hash in the error?

Yes, but i avoid strictly to introduce new functionality whilst fixing things (usually I don't touch even the whitespace). Thus I adopted for now the existent code. Less stress and makes review easier.

In a next iteration I'll go for "refine the relevant debug output / error messages" as a whole (e.g. print out block-no/hash in a uniform way, not only for this error, but for others & debug-log output, too ).

Reply:
Fair enough, thank you for your work here! :)